README.md

```diff
@@ -1 +1,19 @@
 # hw01
+
+##Background Information:
+My name is **_Natalia Joseph_**
+![Photo of me](https://images.app.goo.gl/iYvQDpmX9QDD6icQ6)
```

Comment:
Also, the image you attached here did not show up when I clicked the link. Maybe there was an error with the url.

Reply:
I think you meant

```
![](https://media-exp1.licdn.com/dms/image/C4E03AQE6MRrhNkc6Aw/profile-displayphoto-shrink_200_200/0?e=1607558400&v=beta&t=EBXwCHFifLwz0YwD823-sJQ0x_gIcwGfsz-AtMQ78ZE)
```

![](https://media-exp1.licdn.com/dms/image/C4E03AQE6MRrhNkc6Aw/profile-displayphoto-shrink_200_200/0?e=1607558400&v=beta&t=EBXwCHFifLwz0YwD823-sJQ0x_gIcwGfsz-AtMQ78ZE)